src/org/jitsi/util/RTPUtils.java

```diff
@@ -23,6 +23,40 @@
  */
 public class RTPUtils
 {
+    /**
+     * A <tt>Comparator</tt> implementation for RTP sequence numbers.
+     * Compares <tt>a</tt> and <tt>b</tt>, taking into account the wrap at 2^16.
+     *
+     * IMPORTANT: This is a valid <tt>Comparator</tt> implementation only if
+     * used for subsets of [0, 2^16) which don't span more than 2^15 elements.
+     *
+     * E.g. it works for: [0, 2^15-1] and ([50000, 2^16) u [0, 10000])
+     * Doesn't work for: [0, 2^15] and ([0, 2^15-1] u {2^16-1}) and [0, 2^16)
+     */
+    public static final Comparator<? super Integer> seqNumComparator
```

Comment:
Thanks for moving this to the utils class. But we already have RTPUtils.sequenceNumberComparator and I think they implement the same thing.

Reply:
ah! even better :)